package.json

```diff
@@ -37,9 +37,9 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "xo && tap --coverage --reporter=spec --timeout=150 test/*.js",
-    "test-win": "tap --reporter=spec --timeout=150 test/*.js",
-    "coverage": "tap --coverage-report=lcov"
+    "pretest": "rm -rf ./node_modules/.cache",
```

Comment:
OK, I tried `trash-cli` first but it fails if the directory doesn't exist. Was just about to push `rimraf`